tools/hlslpm/github.py

```diff
@@ -0,0 +1,144 @@
+from enum import Enum
```

Comment:
I find it helpful to have a file-level comment that describes the contents of the script at a high level. Even something as simple as "Contains a class that can be used to access github issues and associated unit tests" is a nice way to quickly understand what we should expect to see.

Reply:
Added some.